rlberry/tests/test_agent.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from rlberry.agents import *
+from rlberry.agents.torch import *
+from rlberry.utils import check_rl_agent
+from rlberry.agents.features import FeatureMap
+import numpy as np
+
+
+class OneHotFeatureMap(FeatureMap):
+    def __init__(self, S, A):
+        self.S = S
+        self.A = A
+        self.shape = (S * A,)
+
+    def map(self, observation, action):
+        feat = np.zeros((self.S, self.A))
+        feat[observation, action] = 1.0
+        return feat.flatten()
+
+
+class OneHotLSVI(LSVIUCBAgent):
+    def __init__(self, env, **kwargs):
+        def feature_map_fn(_env):
+            return OneHotFeatureMap(5, 2)  # values for Chain
+
+        LSVIUCBAgent.__init__(
+            self, env, feature_map_fn=feature_map_fn, horizon=10, **kwargs
+        )
```

Comment:
I agree that it might be a good idea to always provide default kwargs for agents, which would avoid wrapping here (but I think it's something for another PR).

Reply:
I will keep it as is but maybe in a futur PR we may want to set a guideline like "all parameters that are not env should have a default" ?

Reply:
Ok! Yes, I think that it'd be a nice guideline! But only for the agents that we add/test in the library (so we don't add any constraint to user-only code).